schemaless/create_uuid_map.py

```diff
@@ -47,9 +49,15 @@ def from_files(cls, schemaless_file, uuid_map_file):
                 # TODO: Refactor the source-specific logic somewhere else.
                 if source == PPTS.NAME:
                     if 'parent' in record:
-                        parents.extend(record['parent'].split(","))
+                        for parent in record['parent'].split(","):
+                            prj = prj_to_ppts.get(parent)
```

Comment:
another nit on the naming: "prj" is also a bit confusing, can this be renamed to "parent_fk"? and maybe "parent" can be  "parent_id"